Documentation/ldap-connector.md

```diff
@@ -0,0 +1,99 @@
+# Authentication through LDAP
+
+## Overview
+
+The LDAP connector allows email/password based authentication, backed by a LDAP directory. All the configuration parameters required by the connector are held in [Config struct](../connector/ldap/ldap.go).
```

Comment:
We don't need to link to source code here. Probably just cut the second sentence.